pdns/dnsrulactions.hh

```diff
@@ -913,6 +918,8 @@ private:
   string d_fname;
   FILE* d_fp{0};
   bool d_binary{true};
+  bool d_append{false};
+  bool d_buffered{true};
```

Comment:
It looks like we don't need to keep `d_append` and `d_buffered` around, since they seem to be only used in the constructor.